lambify/bin/_bootstrap

```diff
@@ -1,20 +1,6 @@
 #!/bin/sh
 set -e
 
-SAM_BUCKET_NAME_FILE=".bucket-name"
-
-if [ ! -f $SAM_BUCKET_NAME_FILE ]; then
-  echo '== Creating SAM deployment bucket =='
-  rand=$(ruby -r 'securerandom' -e 'puts(SecureRandom.hex[1,10])')
-  name="lamby-{% include "_cctmp/dash_name.txt" %}-${rand}"
-  echo -n "$name" > "$SAM_BUCKET_NAME_FILE"
-  aws s3 mb "s3://${name}"
-else
-  echo '== Using SAM deployment bucket =='
-  name=$(cat $SAM_BUCKET_NAME_FILE)
-  echo $name
-fi
-
```

Comment:
This was not needed for this work but I was here and made the change. Basically AWS SAM fixed a bug that required S3 options to be in place even when containers were needed. S3 with SAM is only needed for ZIP package format. 

I made a note in this issue (https://github.com/customink/lamby-cookiecutter/issues/9) about both but eventually we will be able to drop the ECR in `./bin/_bootstrap` too since SAM will support creating that automatically as well.